Makefile

```diff
@@ -47,6 +47,11 @@ load-hpoa-by-pub:
 load-maxoa:
 	$(CURATE) -v index --batch-size 10 -V maxoa  -c maxoa -m openai: data/maxoa.tsv
 
+# note: assumes local checkout in sibling directory;
+# in future it should pull this from the web
+load-rdp:
+	$(CURATE) index --view reusabledata -c datasets_rdp  --source-locator ../reusabledata/data-sources/ -m openai:
```

Comment:
I would note that this will likely include data sources that are abandoned and/or not public for various reasons. You should either filter on `status: complete` or use the final official public data product: https://reusabledata.org/data.json . I know you likely prefer the YAML, but the latter is "officially" supported.

Reply:
Thanks! Now uses the json